packages/server/graphql/mutations/helpers/bootstrapNewUser.ts

```diff
@@ -62,8 +62,8 @@ const bootstrapNewUser = async (newUser: User, isOrganic: boolean, searchParams?
       highestTier: tier,
       isPatient0
     },
-    anonymousId: segmentId
```

Comment:
Can you double check today we are sending `anonymousId` from Segment to Amplitude and Amplitude is using that as the `deviceId`?

I checked your account, in PG your account has the segmentId `f8774b9d-xxxx...` but that doesn't match any deviceId in [Amplitude](https://app.amplitude.com/analytics/parabol/project/224689/search/amplitude_id%3D314837844648)?

Reply:
We're not sending it today.

What this will do is add the `segmentId` as an additional `deviceId` in Amplitude, allowing us to merge accounts and have more of an audit trail.  Once I get client tracking setup, I need to confirm that we're not leaving anonymous `deviceIds` hanging that should be tied to a `userId`, but am planning to address that in a follow-on PR.